test/linter/test-obsolete.js

```diff
@@ -0,0 +1,110 @@
+/* This file is a part of @mdn/browser-compat-data
+ * See LICENSE file for more information. */
+
+import chalk from 'chalk-template';
+import bcd from '../../index.js';
+const { browsers } = bcd;
+
+const exceptions = [
+  'api.VRDisplay.hardwareUnitId',
+  'api.VREyeParameters.recommendedFieldOfView',
+  'api.VREyeParameters.renderRect',
+  'api.VRFieldOfView.VRFieldOfView',
+  'api.VRPose.hasOrientation',
+  'api.VRPose.hasPosition',
+  'css.types.length.lh',
+  'css.types.length.rlh',
+  'http.headers.Cache-Control.stale-if-error',
+  'http.headers.Feature-Policy.layout-animations',
+  'http.headers.Feature-Policy.legacy-image-formats',
+  'http.headers.Feature-Policy.oversized-images',
+  'http.headers.Feature-Policy.unoptimized-images',
+  'http.headers.Feature-Policy.unsized-media',
+  'svg.attributes.presentation.color-rendering',
+  'svg.elements.view.zoomAndPan',
+];
+
```

Comment:
I think we should define the `warningDate` and `errorDate` variables up top rather than over and over in the function calls.  I also think we should use `setFullYear()`, because it helps clarify what is getting bumped:

```suggestion

let errorTime = new Date();
errorTime.setFullYear(errorTime.getFullYear() - 2.5);
let warningTime = new Date();
warningTime.setFullYear(warningTime.getFullYear() - 2);

```